app/git-info.js

```diff
@@ -3,6 +3,33 @@
 const fs = require('fs')
 const path = require('path')
 
+function packedRefsParse(gitDir, ref) {
```

Comment:
Sorry my comment about adding a comment was unclear. Could you note why we sometimes have to look in packed refs?